docs/backend-api/backend-api-installation.md

```diff
@@ -0,0 +1,55 @@
+# Backend API Installation
+
+*Please follow these instructions with clean git working tree (you can verify with `git status`). If the installation fails, you can easily revert all changes and start again.*
+
+*If you installed the Shopsys Framework with docker please run all commands in the `php-fpm` docker container*
+
+Require `shopsys/backend-api` package in the `composer.json` or run
+
+```bash
+composer require shopsys/backend-api
```

Comment:
This will not work until we release `v8.0.0`. Composer will fail:
```
  [InvalidArgumentException]                                                                                           
  Could not find a version of package shopsys/backend-api matching your minimum-stability (stable).
  Require it with an explicit version constraint allowing its desired stability.   
```
I am OK with this behavior as the error message is pretty explanatory and issues like this can be handled easily. Compatibility with the user's current version will be handled by Composer as `shopsys/backend-api` is dependent on `shopsys/framework` in a particular version (or version range).